src/multi_select.coffee

```diff
@@ -98,12 +107,27 @@ Ember.Widgets.MultiSelectComponent = Ember.Widgets.SelectComponent.extend
     @set 'values', Ember.A [] unless @get('values')
 
   deletePressed: (event) ->
-    if event.target.selectionStart == 0
+    if event.target.selectionStart is 0 and event.target.selectionEnd is 0
       @removeSelectItem(@get('selections.lastObject'))
 
   removeSelectItem: (item) ->
     @get('selections').removeObject item
 
+  focusIn: (event) ->
+    @addBorderMultiSelectContainer()
```

Comment:
I would probably set a boolean ( `hasFocus` ) to `yes` / `no` and use `bind-attr` in the template to toggle the class